libpod/container.go

```diff
@@ -569,6 +569,11 @@ func (c *Container) RuntimeName() string {
 	return c.config.OCIRuntime
 }
 
+// SetPidFile set pidFile for save container process id
+func (c *Container) SetPidFile(pidFile string) {
```

Comment:
We can't do runtime changes like this - this needs to be set on initial container creation via an option (see `libpod/options.go`)

Reply:
use `libpod/options.go`  to set pidfile